sdk/nodejs/project.json

```diff
@@ -45,6 +45,24 @@
           }
       ]
     },
+    "compile-pb": {
```

Comment:
why does node need its own build step for this? it should be exported from the assembly script lib?